cime_config/machines/config_machines.xml

```diff
@@ -4124,15 +4124,15 @@
     <BATCH_SYSTEM>slurm</BATCH_SYSTEM>
     <SUPPORTED_BY>e3sm</SUPPORTED_BY>
     <MAX_TASKS_PER_NODE>256</MAX_TASKS_PER_NODE>
-    <MAX_MPITASKS_PER_NODE>64</MAX_MPITASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE>128</MAX_MPITASKS_PER_NODE>
```

Comment:
@mark-petersen, the issue you pointed out on Slack should be fixed here.

Reply:
Yes, this should definitely be 128 for chicoma-cpu. Thanks.